proto/kyve/multi_coin_rewards/v1beta1/events.proto

```
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package kyve.multi_coin_rewards.v1beta1;
+
+import "gogoproto/gogo.proto";
```

Buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)